users/tests.py

```diff
@@ -378,6 +346,15 @@ def test_user_is_created(self):
 
         self.assertEqual(created_user, user)
 
+    def test_throws_exception_if_email_already_exists(self):
+        EMAIL = "existing@email.com"
+        PASSWORD = "pass"
```

Comment:
They're just regular variables. 


```suggestion
        email = "existing@email.com"
        password = "pass"
```